packages/lesswrong/components/tagging/TagPreviewDescription.tsx

```diff
@@ -17,13 +21,17 @@ const TagPreviewDescription = ({tag, classes}: {
   const { ContentItemBody } = Components;
 
   if (!tag) return null
-  const highlight = truncate(tag.description?.htmlHighlight, 1, "paragraphs", "")
+  
+  const highlight = truncate(tag.description?.htmlHighlight, 1, "paragraphs",
```

Comment:
Alas, this currently creates a "read more" link, that when clicked, doesn't actually do anything. Quite a frustrating experience.